### ISSUE TITLE
Still no raw image was found

### ISSUE THREAD

**Issue by SA7BNT** [gvandela/MeteorRpiReceiver#12]
Can some one help me out.
Running a couple of passes, but every time i get " no raw image was found "

![bildschirmfoto 2018-11-04 um 20 21 26](https://user-images.githubusercontent.com/35367117/47968810-3f223c00-e06f-11e8-85db-a11def4a5238.png)
 And it seems not to be the 0Bytes size issue.
![bildschirmfoto 2018-11-04 um 20 22 33](https://user-images.githubusercontent.com/35367117/47968832-814b7d80-e06f-11e8-8e56-ad310cfd5b1d.png)
Some idee ??

**Comment by SA7BNT** [comment]
Ok, it seems that there is no input audio. I tested an .s file from K2SDR 
https://www.dropbox.com/sh/wwgir2e64ubdldb/AADHzKb2Z5WUckTRbDYfGMJ1a?dl=0
Here i´m getting an output bmp when im running medet with the K2SDR file. 
How can i now test my input audio?

**Comment by SA7BNT** [comment]
![bildschirmfoto 2018-11-05 um 11 04 46](https://user-images.githubusercontent.com/35367117/47991525-adf0ab00-e0ea-11e8-8854-47c22ce91cb4.png)

**Comment by gvandela** [comment]
"The usb_claim_interface error -6" can be fixed by blacklisting the linux driver, it seems. A Google search result: [(https://www.rtl-sdr.com/forum/viewtopic.php?t=111)]

**Comment by SA7BNT** [comment]
Thanks for sharing this with me. Have add this and usb_claim_interface error -6 is not coming up more.
I will now wait for my best pass today @19:33 77 degrees elevation.

**Comment by SA7BNT** [comment]
Still no raw was found. Don´t know what the issue can be. 
![bildschirmfoto 2018-11-05 um 21 46 09](https://user-images.githubusercontent.com/35367117/48025737-86c6c780-e144-11e8-9619-a753ae1b333d.png)
And that was 77degrees elevation. :(

**Comment by gvandela** [comment]
I understand your frustration. I spent some time during my holidays in SF to capture some images and was utterly unsuccessful.
I read about a Linux piece of software by 5b4az.org. Maybe you can check that out. Should give you some visual feedback on the signal you're receiving as well.

**Comment by SA7BNT** [comment]
Yes, i read about it yesterday. Will give it a try. But a little bit frustrated if you don´t know what the issue is. Here in your project. :(

**Comment by gvandela** [comment]
If it works out with the other software, at least we'll know your setup is fine. Then we can keep digging ;)
